deployment/ocp/cloudigrade.yml

```diff
@@ -8,7 +8,123 @@ objects:
 - kind: DeploymentConfig
   apiVersion: v1
   metadata:
-    name: cloudigrade
+    name: ${NAME}-celery
+  spec:
```

Comment:
You do not have a a [deployment strategy](https://docs.openshift.com/dedicated/dev_guide/deployments/deployment_strategies.html) defined. Is celery compatible with a rolling deployment strategy (the default)? Or would using recreate here be better?

If you'd like to switch to using a recreate simply add 
```
strategy:
  type: Recreate
```
between the metadata and spec.

Reply:
Does `strategy` belong *in* the `spec`, not as a peer? That's where it lives for the other `DeploymentConfig`s.

Reply:
That's correct, my mistake!